jobs/nats/spec

```diff
@@ -49,3 +64,32 @@ properties:
   nats.tls.timeout:
     description: Timeout (in seconds) for TLS handshake
     default: 30
+
+  #
+  # Bosh Director options (to get the list of managed VMs)
+  #
+
+  director.address:
+    description: Address of the Bosh Director to connect to
+  director.port:
+    description: Port of the Bosh Director to connect to
+    default: 25555
+  nats.director_account.user:
```

Comment:
Is there also a PR for bosh-deployment which adds all these new properties? Asking because we probably want to make sure some coördination happens around making sure these things get released at the same time.

Reply:
@rkoster there's no PR yet, however, our development tasks for NATs 2 include one to modify bosh-deployment. As you mention coordination needs to happen between the creation of the BOSH release with NATs 2 and the changes on bosh-deployment.